### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by george-angel**

### CODE REVIEW

File: kubectl/client.go [hunk 1]
```diff
@@ -168,7 +182,10 @@ func (c *Client) applyKustomize(ctx context.Context, path string, options ApplyO
 	}
 	displayArgs = append(displayArgs, "apply", "-f", "-")
 	displayArgs = append(displayArgs, options.Args()...)
-	kubectlCmd := exec.Command("kubectl", displayArgs...)
+
+	// Add opts that are specific to this client
+	args := append(c.KubeCtlOpts, displayArgs...)
+	kubectlCmd := exec.Command(c.KubeCtlPath, args...)
```
Comment (ribbybibby) [inline kubectl/client.go:188]:
```suggestion
	// Add opts that are specific to this client
	displayArgs = append(c.KubeCtlOpts, displayArgs...)
	kubectlCmd := exec.Command(c.KubeCtlPath, displayArgs...)
```
Is there any utility in defining a new `args` variable? I'd just append to the existing `displayArgs`.

File: main.go [hunk 2]
```diff
@@ -164,7 +164,7 @@ func main() {
 		Clock:          clock,
 		DryRun:         *fDryRun,
 		KubeClient:     kubeClient,
-		KubectlClient:  kubectlClient,
+		KubeCtlClient:  kubeCtlClient,
```
Comment (ribbybibby) [inline main.go:167]: Is `KubeCtl`/`kubeCtl` the right casing? In my mind, `kubectl` is one word, even if it's a portmanteau of 'kubernetes' and 'control'.

Comment (george-angel) [reply main.go:167]: Agree - but most of "kubebuilder" is using this casing, so its easier to work being consistent with the lib.

Comment (ribbybibby) [reply main.go:167]: Okay, makes sense :+1:

**Review by ffilippopoulos — APPROVED** [review]

**Pull request merged**